visidata/main.py

```diff
@@ -56,7 +56,7 @@ def duptty():
     try:
         fin = open('/dev/tty')
         fout = open('/dev/tty', mode='w')
-        stdin = open(os.dup(0), encoding=vd.options.getonly('encoding', 'global', 'utf-8'))
+        stdin = open(os.dup(0), mode='rb')
```

Comment:
We can't make stdin binary or people won't be able to use text loaders.

Reply:
i cannot use the open_bytes with stdin. it used the BytesIOWrapper which is really hacky, but it also doesnt work once i use io.TextIOWrapper (read1 method doesnt exists on textiowrapper). i fixed it by opening stdin twice.